src/tests/epics/getStoreLocations.epic.test.js

```diff
@@ -0,0 +1,37 @@
+import { ActionsObservable } from "redux-observable"
+import "rxjs"
+import { Observable } from "rxjs/Observable"
+import "rxjs/add/observable/of"
+
+import { getStoreLocations } from "../../redux/epics/getStoreLocations.epic"
+import {
+  getStoresByProductIDs,
+  sendStoresByProductIDs,
```

Comment:
Hello... this throws a warning, i.e. sendStoresByProductIDs is defined but never used